html-css/week1/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
+</head>
+<body>
+    <nav class="navbar">
+        <div class="container">
+            <ul class="navbar-links">
+            <li><a href="#home">HOME</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#portafolio-section">Portafolio</a></li>
+            <li><a href="#contact">Contact</a></li>
+            </ul> 
+        </div>
+
+    </nav>  
+     <main>
+     <section id="home">
+        <div class="mylogo">
+            <img src="profile-pic.png" class="mylogo" alt="Logo" />
+            <h2>Stefany Romero</h2>
+        </div>
+        <div class="home-text">
+            <h2>Welcome to my Portafolio</h2>
+            <p>Im Stefany, Currently learning HTML and CSS in Hack your future bootcamp.</p>
+        </div>
+     </section>
+
+     <section id="about">
+        <div class="about-text">
+            <h2>About me</h2>
+            <p>Hi there, Im Stefany Romero, Im from Colombia and I have been in Dk for the last 5 years.
+                Currently im in Hack Your Future bootcamp where Im developing new skills as a web developer,
+                Now Im building my portfolio, here you can see how I improving my tecnical skills.
+            </p>
+        </div>
+     </section>
+     <div class="all-skills">
```

Comment:
consider having  the all-skills and skills divs in a <section> element for better semantic structure like you did with the about me section.